src/lib/WidgetApi.ts

```diff
@@ -1,12 +1,29 @@
 import config from '@/config';
 
+interface WidgetApiRequestOptions {
+  method?: 'get' | 'post';
+  body?: FormData;
+}
+
 export default class WidgetApi {
-  static async request<T>(url: string): Promise<T> {
-    const response = await fetch(url, {
+  static async request(
+    url: string,
+    options: WidgetApiRequestOptions,
+  ): Promise<any> {
```

Comment:
Why the change from using `T`? If it's just that we don't want to type here, we should use `unknown`.

Reply:
Copilot did this 🙄